app/views/pages/derivatives.html.erb

```diff
@@ -0,0 +1,160 @@
+<% provide(:title, 'Derivatives') %>
+
+<% cache do %>
+<div class="container">
+  <h1 class="about_h1">Using the Archives Unleashed Cloud Derivative Files</h1>
+  <p class="about_p">One of the main features of the Archives Unleashed Cloud is the creation of <strong>derivative files</strong>. These are datasets that provide information about your web archives and can be used by researchers in place of the original WARC files.</p>
+  <p class="about_p">Once analytics have been run on a collection, as discussed in the <a href="/documentation">AUK documentation</a>, you will be able to view and access a series of four  derivative files.</p>
+  <div class="card-deck">
+    <div class="card">
+      <div class="imageContainer"> 
+        <%= image_tag("DER-gexf.png", alt: "Gephi File Output", class:"card-img-top")%>
+      </div>
+      <div class="card-body">
+        <h5 class="card-title"><strong>Gephi Output</strong></h5>
+      </div>
+      </div>
+    <div class="card">
+      <div class="imageContainer">
+        <%= image_tag("DER-graphml.png", alt: "Raw Network File Output", class:"card-img-top")%>
+      </div>
+      <div class="card-body">
+        <h5 class="card-title"><strong>Raw Nework Output</strong></h5>
```

Comment:
typo: should be "Network"

Just to make everything work a bit more smoothly on a smaller screen, I would suggest removing the repetitive "Output" from the cards. So just "Network"